recipes/isl/build.sh

```diff
@@ -0,0 +1,4 @@
+./configure --prefix="$PREFIX" --with-gmp-prefix="$PREFIX"
```

Comment:
So, I would add an `if` statement before the configure to check the platform and set `CC` and `CXX`. It might look something like [this]( https://github.com/conda-forge/vigra-feedstock/blob/master/recipe/build.sh ), but not necessarily exactly.